lib/fluent/plugin/bigquery/writer.rb

```diff
@@ -45,10 +45,9 @@ def create_table(project, dataset, table_id, record_schema)
             return
           end
 
-          reason = e.respond_to?(:reason) ? e.reason : nil
```

Comment:
This logic was added in https://github.com/kaizenplatform/fluent-plugin-bigquery/pull/65, but `Google::Apis::Error` doesn't have `reason` method.
cf. https://github.com/googleapis/google-api-ruby-client/blob/0.24.3/lib/google/apis/errors.rb

I think the class which has `reason` method is `Google::Apis::BigqueryV2::ErrorProto`.
cf. https://github.com/googleapis/google-api-ruby-client/blob/0.24.3/generated/google/apis/bigquery_v2/classes.rb#L2433-L2445